src/data/protocols/db/guardian/add-guardian-repository.ts

```diff
@@ -3,7 +3,7 @@ export interface AddGuardianRepository {
 }
 
 export namespace AddGuardianRepository {
-  export interface Params {
+  export type Params = {
```

Comment:
the idea is to put them all in types or in interfaces. I opted for types because some parameters cannot be interfaces and it ended up being kind of strange to have type and interface to represent params and result.

Reply:
Got it. I'm also choosing to do this in the project I'm using to practice clean architecture